packages/core/src/external/carequality/ihe-gateway-v2/ihe-gateway-v2-logic.ts

```diff
@@ -123,10 +127,12 @@ export async function createSignSendProcessDRRequests({
     patientId,
     cxId,
   });
+  const s3Utils = new S3Utils(region);
```

Comment:
per our discussion in Slack re: gw-v2, if AWS_REGION is a required env variable, then we don't need to set the region explicitly, as it will inherit the region from the env variable (https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/setting-region.html#setting-region-environment-variable).

Reply:
Right - but as of now S3utils requires the region: https://github.com/metriport/metriport/blob/01e0d945e6967a01eb7315b5b666ec8a772be8e3/packages/core/src/external/aws/s3.ts#L49

Happy to make this refactor in a follow up small PR but given that there's a refactor of unrelated code required here, I am gonna try to keep it simple and not introduce that into this PR. Wdyt?